baseapp-wagtail/baseapp_wagtail/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.urls import include, path
+from wagtail import urls as wagtail_urls
+from wagtail.admin import urls as wagtailadmin_urls
+from wagtail.documents import urls as wagtaildocs_urls
+
+from .api.router import api_router
+
+urlpatterns = [
+    path("cms/", include(wagtailadmin_urls)),
+    path("documents/", include(wagtaildocs_urls)),
+    path("api/v2/", api_router.urls),
+    # TODO: Find a way to disable the Wagtail pages. If we remove this line, the system won't
+    # generate the url_path.

```

Comment:
can you expand on this comment please? It won't generate the `url_path` for the page?

Reply:
Yeah. Like the path `path("", include(wagtail_urls)),` will render the wagtail pages using jinja2. Without it set, the system won't provide a final path for the pages and we need it to generate the headless pages. 
The solution would be to override the function that generates the url_path to use the FRONT_HEADLESS_URL. For now, I just activated it. It's harmless. But ideally we have to solve the TODO.